timemachine/fe/frames.py

```diff
@@ -1,14 +1,16 @@
-from typing import List, Tuple, Generator
+from timemachine.fe import estimator, estimator_abfe
+from typing import List, Tuple, Generator, Union
 
 
+SimulationResult = Union[estimator.SimulationResult, estimator_abfe.SimulationResult]
```

Comment:
Nice -- `"SimulationResult"` was used instead of `SimulationResult` with [exactly this intent](https://github.com/proteneer/timemachine/pull/528#discussion_r746874882)!

(nit: only remaining line to update looks like [L6](https://github.com/proteneer/timemachine/blob/6fec4c23000a0f4170fee038cbcb2a1d385ba06c/timemachine/fe/frames.py#L6))

Reply:
Ah, thanks! Fixed in [f15f731](https://github.com/proteneer/timemachine/pull/596/commits/f15f7313271b400ee416ed4889e33561d12a2038).

Reply:
Actually thinking through this more, it seems like it'd be more correct to use `Any` here, since the implementation doesn't assume anything about `SimulationResult`. I changed it in [f7a4e32](https://github.com/proteneer/timemachine/pull/596/commits/f7a4e328660aac1bbea7e2f952d0fe4b51b244ba), but willing to be convinced otherwise